Meadowlark-js/tests/http/local.ownership-tests.http

```diff
@@ -155,3 +155,32 @@ DELETE http://localhost:3000/local/v3.3b/ed-fi/educationContents/YKe0uvuOUdWgG2M
 ### Delete 2nd resource using the same authorization header that created the record - expect success.
 DELETE http://localhost:3000/local/v3.3b/ed-fi/educationContents/1vvBcOeM1WsZkfwgE-edz-dPgkHJY7uyBrEkuQ
 {{auth_header_2}}
+
```

Comment:
The test on line 73 is giving me a rather unusual response with PostgreSQL as the backend, but not MongoDB:

```
HTTP/1.1 400 Bad Request
content-type: application/json
content-length: 28
Date: Wed, 27 Jul 2022 22:08:09 GMT
Connection: close

{
  "message": "Malformed body"
}
```

I doubt that it is due to any changes in this PR. I'm going to move past that and create a separate bug ticket.